tests/tests.rs

```diff
@@ -1014,3 +1015,73 @@ fn reject_import_instance_memory() -> Result<()> {
         "#,
     )
 }
+
+#[test]
+fn accept_bulk_memory_copy() -> Result<()> {
+    run_wat(
+        &[],
+        ('h' as i32) + ('w' as i32),
+        r#"
+            (module
+              (memory $memory (data "hello, wizer!"))
+              (func (export "wizer.initialize")
+                i32.const 42 ;; dst
+                i32.const 0  ;; src
+                i32.const 13 ;; size
+                memory.copy)
+              (func (export "run") (result i32)
+                i32.const 42
+                i32.load8_u
+                i32.const 42
+                i32.load8_u offset=7
+                i32.add))
+        "#,
+    )
+}
+
+#[test]
+fn accept_bulk_memory_fill() -> Result<()> {
+    run_wat(
+        &[],
+        77 + 77,
+        r#"
+            (module
+              (memory 1)
+              (func (export "wizer.initialize")
+                i32.const 42 ;; dst
+                i32.const 77 ;; value
+                i32.const 13 ;; size
+                memory.fill)
+              (func (export "run") (result i32)
+                i32.const 42
+                i32.load8_u
+                i32.const 42
+                i32.load8_u offset=7
+                i32.add))
+        "#,
+    )
+}
+
+#[test]
+fn accept_bulk_memory_init() -> Result<()> {
```

Comment:
Can you add `reject_` tests for each of the mutating instructions from bulk-memory? I see we only have one test for `table.copy` right now, but we should also reject:

* `table.set`
* `table.init`
* `data.drop`
* `elem.drop`

and I think that is all of them?

Reply:
There's also `table.fill`, but I don't _think_ it's possible to test that without using `table.get` first, so any such test would error out before it even got to the `table.fill`.  Same story for `table.set`.

I added a `reject_table_get_set` function and _could_ add a `reject_table_get_fill` one as well, but they would both result in the same error.

Reply:
The validation as it is right now actually allows `table.get` since it doesn't mutate anything, so I don't think the two tests would be redundant.

Reply:
Hmm, I'm getting the same "reference types support is not enabled" error in both cases.  Here's the `table.fill` test:

```

#[test]
fn reject_table_fill() -> Result<()> {
    let result = run_wat(
        &[],
        42,
        r#"
(module
  (table 3 funcref)

  (func $f (result i32) (i32.const 0))
  (func $g (result i32) (i32.const 0))
  (func $h (result i32) (i32.const 0))

  (elem $elem $f $g $h)

  (func (export "main")
    i32.const 0
    i32.const 0
    table.get
    i32.const 3
    table.fill)
)
"#,
    );
    assert!(result.is_err());

    let err = result.unwrap_err();
    assert!(err
        .to_string()
        .contains("reference types support is not enabled"),);

    Ok(())
}
```

The test does pass, and I'm happy to add it, but I don't think it's serving its intended purpose.

Reply:
Ah nvm I forgot that `table.get` and `table.set` got pushed into reference types, and aren't part of bulk memory. Good to have these tests as a future proofing though, anyways. Thanks!